### PR TITLE
Fix installation bugs and add Windows installation

### PR DESCRIPTION
**Pull request by jvican**
1. Configure a scoop-based installation for Windows Powershell users.
2. Improve the ergonomics of Windows CMD users.
3. Fix several bugs with the installation script and the nailgun script.
4. Fix a bug in the build when running gradle in Windows.
5. Improve the UX of the installation page by persisting the choices to
   the session storage.

The fixed tickets are linked in the commit messages.

### CODE REVIEW

File: website/static/scripts/build-tools.js [hunk 1]
```diff
@@ -34,6 +34,12 @@
     }
   }
 
+
+  previouslyBuildActive = null;
+  if (typeof(Storage) !== "undefined") {
```
[the remark this answers is not in the document]
Comment (jvican) [reply website/static/scripts/build-tools.js]: 👍 thanks, good to know, will leave it as is to prove how terrible a js developer am, i cargo-culted it from https://www.w3schools.com/html/html5_webstorage.asp

[the remark this answers is not in the document]
Comment (gabro) [reply website/static/scripts/build-tools.js]: LGTM! Can I suggest https://chrome.google.com/webstore/detail/remove-w3schools/gohnadkcefpdhblajddfnhapimpdjkje

[the remark this answers is not in the document]
Comment (jvican) [reply website/static/scripts/build-tools.js]: Hahaha, nice one 👍

**Comment by jvican** [comment]
I'm merging, the Windows CI error seems spurious and this doesn't change the main logic of the server anyway.

**Pull request merged**